Tutorials/pinpull.md

```diff
@@ -0,0 +1,51 @@
+# Pull- Pins
+
+Suppose a pin is configured as an input. If nothing is connected to the pin and the program tries to read the state of the pin, it would be in a 'floating' state i.e an unknown state. To prevent this, a pull-up or a pull-down state is defined. They are often used in the case of Buttons and Switches.
+
+Pins 2-7 on both the Ports are available for interrupts.
```

Comment:
"...available for interrupts" – did you mean to say pull-up and pull-down instead of interrupts?